src/chain/fee.rs

```diff
@@ -113,18 +113,6 @@ impl From<u8> for Gas {
     }
 }
 
-impl From<cosmrs::tx::Gas> for Gas {
-    fn from(g: cosmrs::tx::Gas) -> Gas {
-        g.value().into()
-    }
-}
-
-impl From<Gas> for cosmrs::tx::Gas {
-    fn from(g: Gas) -> cosmrs::tx::Gas {
-        g.value().into()
-    }
-}
```

Comment:
This is redundant now that `Gas` exposed from cosmrs is just an alias for u64 because we already have a `impl From<u64> for Gas`